binary_tree_neighbors.js

```diff
@@ -1,23 +1,29 @@
-// Write an algorithm that associates the neighbors of nodes in a binary tree as
-// the picture below suggests:
-//
-//         a-->null
-//        / \
-//       b-->c-->null
-//      /   / \
-//     d-->e-->f-->null
-//    /       / \
-//   g------>h-->i-->null
-//  / \
-// j-->k-->null
-//
+/**
+ *  Associates the neighbors of nodes in a binary tree as the picture below suggests:
+ * 
+ *             a-->null
+ *            / \
+ *           b-->c-->null
+ *          /   / \
+ *         d-->e-->f-->null
+ *        /       / \
+ *       g------>h-->i-->null
+ *      / \
+ *     j-->k-->null
+ */
 
-// Data structure:
```

Comment:
Do you mind replacing the comments you propose to remove with a definition of the `node` type via jsdoc's `@typedef` directive?